cli/lib/kontena/cli/services/exec_command.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'services_helper'
+
+module Kontena::Cli::Services
+  class ExecCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+    include ServicesHelper
+
+    parameter "NAME", "Service name"
+    parameter "CMD ...", "Command"
+
+    option ["-i", "--instance"], "INSTANCE", "Exec on given numbered instance, default first running" do |value| Integer(value) end
+    option ["-a", "--all"], :flag, "Exec on all running instances"
+    option ["--shell"], :flag, "Execute as a shell command"
+
+    requires_current_master
+    requires_current_grid
+
+    # Exits if exec returns with non-zero
+    def exec_container(container)
+      if shell?
+        cmd = ['sh', '-c', cmd_list.join(' ')]
+      else
+        cmd = cmd_list
+      end
+
+      stdout, stderr, exit_status = client.post("containers/#{container['id']}/exec", {cmd: cmd})
+
+      STDOUT.puts stdout unless stdout.empty?
+      STDERR.puts stderr unless stderr.empty?
+      exit exit_status if exit_status != 0
```

Comment:
Thinking this could output which container failed, if we're running with `--all`?